features/steps/example_resources.json

```diff
@@ -0,0 +1 @@
+{"count":2,"resources":[{"uri":"/a/temp/uri","uri_old":"/an/old/uri","content_type":"api_dataset_landing_page","cdid":"ASELECTIONOFNUMBERSANDLETTERS123","dataset_id":"ASELECTIONOFNUMBERSANDLETTERS456","edition":"a temporary edition","meta_description":"a temporary description","release_date":"0001-01-01T00:00:00Z","summary":"a temporary summary","title":"a temporary title","topics":["a","b","c","d"],"language":"string","survey":"string","canonical_topic":"string","cancelled":true,"finalised":true,"published":true,"date_changes":["a change_notice","a previous_date"],"provisional_date":"October-November 2024"},{"uri":"/another/temp/uri","uri_old":"/another/old/uri","content_type":"api_dataset_landing_page","cdid":"ASELECTIONOFNUMBERSANDLETTERS789","dataset_id":"ASELECTIONOFNUMBERSANDLETTERS101112","edition":"another temporary edition","meta_description":"another temporary description","release_date":"0001-01-01T00:00:00Z","summary":"another temporary summary","title":"another temporary title","topics":["a","b","e","f"],"language":"string","survey":"string","canonical_topic":"string","cancelled":true,"finalised":true,"published":true,"date_changes":["a change_notice","a previous_date"],"provisional_date":"October-November 2024"}],"limit":20,"offset":0,"total_count":2}
```

Comment:
I did format it originally, for that reason, but then it caused the component test to fail because it was different from the result. So I would prefer not to format it for the component tests (as it would make it difficult to compare it to the result). The ones that get used for the endpoint itself should definitely be formatted though.